eval/mt_bench.py

```diff
@@ -79,10 +79,22 @@ def fetch_secret(secret_name, keys):
         print("Eval Judge secret data retrieved.")
 
     judge_ca_cert_path = os.getenv("JUDGE_CA_CERT_PATH")
-    use_tls = os.path.exists(judge_ca_cert_path) and (
+    dsp_ca_cert_path = os.getenv("SSL_CERT_FILE")
+    dsp_ca_cert_dir = os.getenv("SSL_CERT_DIR")
+
+    dsp_cert_dir_defined = dsp_ca_cert_dir is not None
+    dsp_ca_exists = os.path.isfile(dsp_ca_cert_path) and (
```

Comment:
Could you check if `dsp_ca_cert_path` is defined before trying to read the file at its path?